python/src/aiconfig/editor/client/src/components/prompt/PromptContainer.tsx

```diff
@@ -190,7 +190,9 @@ export default memo(function PromptContainer({
 
           {prompt.outputs && prompt.outputs.length > 0 && (
             <>
+            <Flex justify="space-between" direction="column">
```

Comment:
What does this look like / is this needed now that the button is moved to the menu?